test/source/tests/flaky.ts

```diff
@@ -128,8 +128,8 @@ export const defineFlakyTests = (testVariant: TestVariant, testWithBrowser: Test
       await webDecryptPage.waitAndClick('@action-decrypt');
       await webDecryptPage.waitForContent('@container-pgp-decrypted-content', subject);
       await webDecryptPage.waitForContent('@container-pgp-decrypted-content', 'flowcrypt.compatibility test footer with an img');
-      await webDecryptPage.waitAll('@container-att-name(small.txt)');
-      const fileText = await webDecryptPage.awaitDownloadTriggeredByClicking('@container-att-name(small.txt)');
+      await webDecryptPage.waitAll('@container-attachment-name(small.txt)');
```

Comment:
Need to make sure appropriate html file was updated too in case it wasn't. Look for `container-att-name`

Reply:
And here's the appropriate failing test: https://flowcrypt.semaphoreci.com/jobs/461a47a3-1ad7-4a99-95ae-9bf992ca3e81

But I can't find `container-att-name` anywhere.

Reply:
Not sure where is it defined, either. You can switch this back to `container-att-name` in the test.